<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPC</class>
 <widget class="QDialog" name="LPC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LPC声码器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LPC {
border-image: url(:/new/prefix1/background.jpg);
}
#LPC * {
border-image:url();
}</string>
  </property>
  <widget class="QPushButton" name="lpc_exit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lpcfigure">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lpc_lineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgba(255,0,0,0);border-radius:0px;}</string>
   </property>
   <property name="text">
    <string>请选择原始语音</string>
   </property>
  </widget>
  <widget class="QComboBox" name="lpc_comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>bF0C11011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bF0C11022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bF0C11031</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bF0C11041</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bM0C01013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bM0C01023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bM0C01051</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>182</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="lpc_origin">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>原语音</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="lpc_EGG">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EGG 语音合成</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="lpc_speech">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>语音基频 语音合成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
